projects/br_scs/src/Mirza/SupplyChain/API.hs

```diff
@@ -59,17 +59,7 @@ type ServerAPI =
   --                 :> Capture "userId" UserId
   --                 :> Get '[JSON] [Ev.Event]
 -- Event Registration
-  :<|> "event"    :> "objectEvent"
-                  :> ReqBody '[JSON] ObjectEvent
-                  :> Post '[JSON] (EventInfo, Schema.EventId)
-  :<|> "event"    :> "aggregateEvent"
-                  :> ReqBody '[JSON] AggregationEvent
-                  :> Post '[JSON] (EventInfo, Schema.EventId)
-  :<|> "event"    :> "transactionEvent"
-                  :> ReqBody '[JSON] TransactionEvent
-                  :> Post '[JSON] (EventInfo, Schema.EventId)
-  :<|> "event"    :> "transformationEvent"
-                  :> ReqBody '[JSON] TransformationEvent
+  :<|> "event"    :> ReqBody '[JSON] Ev.Event
```

Comment:
(Future): can we use the XML parsing code we have to also support request bodies as XML (EPCIS 1.2)?

Reply:
We absolutely can. I suspect it will be fairly straightforward to plug the parser in.